Makefile

```diff
@@ -34,11 +31,14 @@ test: lint vet
 watch: bootstrap
 	goconvey
 
-bin/lumogon:
+bin/lumogon: dependencies
```

Comment:
this lengthens the build on every run. If you want a pattern to only run when needed see https://github.com/garethr/kubeval/blob/master/Makefile#L34-L40

Reply:
Does that pattern actually work for you tho?

Reply:
nwm, problem between keyboard and chair